setup.py

```diff
@@ -52,18 +52,14 @@ def install():
                      'Operating System :: MacOS :: MacOS X',
                      'Topic :: Education',
                      'Programming Language :: Python',
-                     'Programming Language :: Python :: 3.5',
                      'Programming Language :: Python :: 3.6',
                      'Programming Language :: Python :: 3.7',
                      'Programming Language :: Python :: 3.8'],
         packages=find_packages(exclude=['docs', 'tests']),
         keywords='google translate translator',
         install_requires=[
-            'requests',
+            'httpx==0.13.3',
```

Comment:
Why using a so specific version of httpx ?

Reply:
Httpx seems using semantic versioning and still in beta, so it may break compatability on future major or minor releases.

https://github.com/encode/httpx/blob/master/README.md#L18